content/3_Silver/Flood_Fill.mdx

```diff
@@ -723,6 +723,68 @@ def floodfill(r: int, c: int):
 		frontier.append((r, c + 1))
 
 
+room_num = 0
+for row in range(row_num):
+	for col in range(col_num):
+		if not visited[row][col] and grid[row][col] == '.':
+			floodfill(row, col)
+			room_num += 1
+print(room_num)
+```
+
+</PySection>
+
+</LanguageSection>
+
+<!-- Below is a python specific optimization. -->
+<LanguageSection>
+
+<CPPSection>
+<!-- Add a section for a different language so the text below only shows for python. -->
+</CPPSection>
+
+<PySection>
+
+## Python Optimization: Padding
```

Comment:
i'd add this to the previous section instead of making a new one

Reply:
So just remove the ## section header?

Reply:
uh no like merge the `PySection` and `LanguageSection` tags